src/ListGroup.js

```diff
@@ -21,6 +21,22 @@ const propTypes = {
    */
   variant: PropTypes.oneOf(['flush', null]),
 
+  /**
+   * Changes the flow of the list group items from vertical to horizontal.
+   * A value of `null` (the default) sets it to vertical for all breakpoints;
+   * `horizontal` sets it go horizontal for all breakpoints, while `horizontal-{sm|md|lg|xl}`
+   * makes the list group horizontal starting at that breakpoint’s `min-width`.
+   * @type {('horizontal'|'horizontal-sm'|'horizontal-md'|'horizontal-lg'|'horizontal-xl')}
+   */
+  layout: PropTypes.oneOf([
+    'horizontal',
+    'horizontal-sm',
+    'horizontal-md',
+    'horizontal-lg',
+    'horizontal-xl',
+    null,
+  ]),
```

Comment:
I think i would change this API to 

```js
horiztonal: PropTypes.oneOf([
    true,
    'sm',
    'md',
    'lg',
    'xl',
    null,
  ]),
```

I think that's probably more inline with how w'eve done this elsewhere in the codebase, e.g. `navbar.expand`

Reply:
Makes sense! Just pushed a commit to the fork that updates that + changes the docs.